recipes/hpack/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "hpack" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "8eec9c1f4bfae3408a3f30500261f7e6a65912dc138526ea054f9ad98892e9d2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - test
+
+about:
+  home: http://python-hyper.org/hpack/
+  license: MIT
+  summary: 'HTTP/2 Header Encoding for Python'
+  dev_url: https://github.com/python-hyper/hpack
+
+extra:
+  recipe-maintainers:
+    - primozgodec
+    - astaric
```

Comment:
@astaric -- are you ok being added as a co-maintainer of the `hpack` recipe on conda-forge?

https://conda-forge.org/docs/recipe.html#maintainer-role